vision/dcgan_mnist/README.md

```diff
@@ -0,0 +1,39 @@
+# Deep Convolutional GAN (DC-GAN)
+
+![dcgan_gen_disc](../dcgan_mnist/output/dcgan_generator_discriminator.png)
+[Source](https://gluon.mxnet.io/chapter14_generative-adversarial-networks/dcgan.html)
+
+## Model Info
+
+A DC-GAN is a direct extension of the GAN, except that it explicitly uses convolutional and transposed convolutions layers in the discriminator and generator, respectively. _The discriminator is made up of strided convolution layers, batch norm layers, and LeakyReLU activations. The generator is comprised of transposed convolutions layers, batch norm layers, and ReLU activations_.
```

Comment:
- I think "transposed convolutional" would read better than "transposed convolutions", especially next to "convolutional".
- What is the intent of italicizing the text at the end?

Reply:
The intent of italicizing is to highlight the layers info. Should I remove it?